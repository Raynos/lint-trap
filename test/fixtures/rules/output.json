[
    {
        "type": "file",
        "file": "func-names-invalid.js",
        "errors": [
            {
                "type": "error",
                "linter": "jscs",
                "file": "func-names-invalid.js",
                "line": 7,
                "column": 19,
                "rule": "disallowSpacesInNamedFunctionExpression",
                "message": "Illegal space before opening round brace"
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "func-names-invalid.js",
                "line": 4,
                "column": 10,
                "rule": "func-names",
                "message": "Missing function expression name."
            }
        ]
    },
    {
        "type": "file",
        "file": "func-names-valid.js",
        "errors": []
    },
    {
        "type": "file",
        "file": "globals-invalid.js",
        "errors": [
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 9,
                "column": 8,
                "rule": "no-undef",
                "message": "'process' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 10,
                "column": 8,
                "rule": "no-undef",
                "message": "'global' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 11,
                "column": 8,
                "rule": "no-undef",
                "message": "'exports' is not defined."
            },
            {
                "type": "error",
                "linter": "eslint",
                "file": "globals-invalid.js",
                "line": 12,
                "column": 8,
                "rule": "no-undef",
                "message": "'console' is not defined."
            }
        ]
    },
    {
        "type": "file",
        "file": "globals-valid.js",
        "errors": []
    }
]
