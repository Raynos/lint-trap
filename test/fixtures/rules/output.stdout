func-names-invalid.js
    error  jscs       7:19   Illegal space before opening round brace
    error  eslint     4:10   Missing function expression name.

globals-invalid.js
    error  eslint     9:8    'process' is not defined.
    error  eslint    10:8    'global' is not defined.
    error  eslint    11:8    'exports' is not defined.
    error  eslint    12:8    'console' is not defined.

